### PR TITLE
Added comments on not set node network/inode condition to unknown.

### LINKED ISSUE #48982: If node is unreachable, also set NetworkUnavailable/InodePressure to Unknown
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:

If node is unreachable, NodeController only set `v1.NodeOutOfDisk`, `v1.NodeMemoryPressure`, `v1.NodeDiskPressure` to unknown; it's better to also set NetworkUnavailable & InodePressure to unknown.

### PR DESCRIPTION
**Pull request by k82cn**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #48982

**Release note**:
```release-note
None
```

### CODE REVIEW

**Comment by gmarek** [comment]
We certainly don't want to change NetworkUnreachable condition, as it's managed on a control plane level, thus the fact that Node is unreachable doesn't impact our ability to monitor it.

As for InodePressure - I think someone said that they want to get rid of that altogether... @kubernetes/sig-node-pr-reviews

File: pkg/controller/node/nodecontroller.go [hunk 1]
```diff
@@ -955,7 +954,16 @@ func (nc *NodeController) tryUpdateNodeStatus(node *v1.Node) (time.Duration, v1.
 		}
 
 		// remaining node conditions should also be set to Unknown
-		remainingNodeConditionTypes := []v1.NodeConditionType{v1.NodeOutOfDisk, v1.NodeMemoryPressure, v1.NodeDiskPressure}
+		remainingNodeConditionTypes := []v1.NodeConditionType{
+			v1.NodeOutOfDisk,
+			v1.NodeMemoryPressure,
+			v1.NodeDiskPressure,
+			// We don't change 'NodeInodePressure' condition, as it'll be removed in future.
```
Comment (feiskyer) [inline pkg/controller/node/nodecontroller.go]: Has `NodeInodePressure` already been deprecated? If yes, add a link to it?

Comment (k82cn) [inline pkg/controller/node/nodecontroller.go]: Good point! Opened https://github.com/kubernetes/kubernetes/issues/49103 to confirm whether we can remove it in 1.8 :).

**Pull request merged**